### DISCUSSION TITLE
Documentation - displaying views

### DISCUSSION THREAD

**Discussion by LEGROSJerome** [mytourbook/mytourbook#1399] [Ideas]
Many thanks for this software. 
In the documentation, I haven't seen anything about managing "views" (screens) and I'm still having a lot of trouble. For example, I can't get 3 views.

Thanks in advance

**Comment by wolfgang-ch** [comment]
These are Eclipse topics which are documented in https://help.eclipse.org/latest/index.jsp?topic=%2Forg.eclipse.platform.doc.user%2Ftasks%2Ftasks-1a.htm&cp%3D0_3_2


![image](https://github.com/user-attachments/assets/c1b220b4-381f-4d3e-97be-8a30aa86b1d3)
